utils/fork-tree/src/lib.rs

```diff
@@ -609,16 +634,17 @@ mod node_implementation {
 		/// when the predicate fails.
 		/// The given function `is_descendent_of` should return `true` if the second hash (target)
 		/// is a descendent of the first hash (base).
-		// FIXME: it would be useful if this returned a mutable reference but
-		// rustc can't deal with lifetimes properly. an option would be to try
-		// an iterative definition instead.
-		pub fn find_node_where<F, P, E>(
+		///
+		/// The returned indexes are from last to first, meaning the last is the least significant
```

Comment:
```suggestion
		/// The returned indices are from last to first, meaning the last is the least significant
```

Reply:
it's not clear what "least significant" means in this comment, but from reading the code i take it to be the earlier index in the traversal path.

Reply:
I updated the docs just calling it the "earliest" and "final" index.